http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/service/leonardoServiceContractModel.scala

```diff
@@ -27,6 +27,7 @@ final case class ListClusterResponse(id: Long,
                                      labels: LabelMap,
                                      jupyterExtensionUri: Option[GcsPath],
                                      jupyterUserScriptUri: Option[GcsPath],
+                                     jupyterStartUserScriptUri: Option[GcsPath],
```

Comment:
do we really intend to expose this in `ListCluster` API as well? If so, this needs to be reflected in swagger doc

Reply:
If you don't care about the field for now, let's leave it out for now? I prefer exposing fields only when needed. `ListCluster` and getCluster API do differ anyway. This won't be the only difference. `GetCluster` joins a few tables, gives back more fields and hence less performant. Even though in this case, returning  `jupyterStartUserScriptUri` doesn't require joining a different table for `ListCluster` today, we might rework our DB schema.